### PR TITLE
Add `Scalafmt` to the `series/2.x` brach

### PR DESCRIPTION
**Pull request by danicheg**
Scalafmt rules are the same as in cats. Also added handy sbt commands for checking/formatting.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,7 +1,5 @@
-addSbtPlugin("com.47deg" % "sbt-microsites" % "1.3.2")
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.5.1")
-addSbtPlugin("com.codecommit" % "sbt-spiewak-sonatype" % "0.20.4")
-
-// Temporary workaround
-addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.5.5")
```
Comment (danicheg) [inline project/plugins.sbt:7]: That came with cherry-picking

**Comment by rossabaker** [comment]
Looks like it needs a workflow regeneration, too.

**Comment by danicheg** [comment]
@rossabaker I've fixed the workflow for `series/2.x` in #255

**Comment by rossabaker** [comment]
Oh, hmm, was it already broken?  I guess a merge should fix this then.

Thanks for sticking with this.  It _will_ make future maintenance easier.

**Comment by danicheg** [comment]
@rossabaker yep, it was. I hope it would finally work after b40b0f3 🤞🏻

**Pull request merged**